python/tvm/driver/tvmc/compiler.py

```diff
@@ -137,6 +137,11 @@ def add_compile_parser(subparsers, _):
         type=parse_pass_list_str,
         default="",
     )
+    parser.add_argument(
+        "--module-name",
+        default="default",
+        help="The output module name.",
```

Comment:
```suggestion
        help="The output module name. Defaults to 'default'.",
```

Reply:
Thanks @leandron . That makes sense. I've changed it now.